src/Apps/Artist/Routes/AuctionResults/Components/AuctionFilters/PriceRangeFilter.tsx

```diff
@@ -0,0 +1,63 @@
+import { Checkbox, Expandable, Spacer } from "@artsy/palette"
+import { PriceRange } from "Components/PriceRange/PriceRange"
+import {
+  CustomRange,
+  DEFAULT_PRICE_RANGE,
+} from "Components/PriceRange/constants"
+import { FC, useMemo } from "react"
+import {
+  useAuctionResultsFilterContext,
+  useCurrentlySelectedFiltersForAuctionResults,
+} from "Apps/Artist/Routes/AuctionResults/AuctionResultsFilterContext"
+import { debounce } from "lodash"
+import { getBarsFromAggregations } from "Components/ArtworkFilter/ArtworkFilters/PriceRangeFilter"
+
+const DEBOUNCE_DELAY = 300
+
+export const PriceRangeFilter: FC = () => {
+  const { setFilter, aggregations } = useAuctionResultsFilterContext()
+  const {
+    priceRange,
+    includeEstimateRange,
+    includeUnknownPrices,
+  } = useCurrentlySelectedFiltersForAuctionResults()
+  const bars = getBarsFromAggregations(aggregations)
+
+  const setFilterDobounced = useMemo(
+    () => setFilter && debounce(setFilter, DEBOUNCE_DELAY),
+    [setFilter]
+  )
+
+  const handlePriceRangeUpdate = (updatedRange: CustomRange) => {
+    setFilterDobounced?.("priceRange", updatedRange.join("-"))
+  }
+
+  return (
+    <Expandable label="Price" expanded borderColor="white100">
```

Comment:
Hm, what's this look like?

Reply:
Yeah I don't think we should disable the border. Not sure why that was specified like that.

Reply:
You are right. fixed.